infra/release/release_note.md

```diff
@@ -1,28 +1,46 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- Add hide-and-show button
+- Show more files
+  - memory trace files
+  - performance trace files
+  - circle log files
+  - extra backend model binary files
+- Open files with dedicated editor/viewers
+  - circle files - 'Circle Viewer'
```

Comment:
https://github.com/Samsung/ONE-vscode/pull/1188/files#r940884148

Let me know if the comment is resolved, I will synchronize this.